netlab/L5/tls_protocol_layer.hpp

```diff
@@ -520,7 +522,7 @@ namespace netlab {
 					this->handshake.body.clientHello.compression_methods[i] = static_cast<CompressionMethod>(*it);
 				}
 				// Extensions
-				extensions_length = *it;
+				extensions_length = deserialize_2_bytes(it);
```

Comment:
From what I recall, extensions length is 1 byte

Reply:
2 bytes